### ISSUE TITLE
[Codelab]: Making Predictions from 2D Data

### ISSUE THREAD

**Issue by buzon7** [tensorflow/tfjs#2644]
Hello! I am going through this [example](https://codelabs.developers.google.com/codelabs/tfjs-training-regression/#0), but my results change with every run using the provided code and parameters. Even the straight line does not work (see the screen) and I cannot fit a higher order polinomials at all with any number of layers (extra credit page). Is this library bugged or out of date? I think the tutorial loads TFJS 1.0.0. I am new to ML, but I know JS better than Python, so was hoping to start from here! Thanks!!!

Edit: Changing index.html to load TFJS 1.5.1 did not change anything - the model still does not converge to an optimal solution! 

[TensorFlow.js Tutorial.pdf](https://github.com/tensorflow/tfjs/files/4023765/TensorFlow.js.Tutorial.pdf)

**Comment by tafsiri** [comment]
Hi. Could you tell us more about what device/os/browser you are running this on? 

The final result does change on different runs but I wasn't able to reproduce your issue with 50 epochs. Though I notice from your screenshot that your loss is still a bit high. You could try upping the epochs to 75 or 100 and seeing how that improves training.

**Comment by buzon7** [comment]
Running on laptop in Windows 10/Opera. 100 epochs is better. I thought the charts were for the default values. Thanks!
